mne/time_frequency/tests/test_tfr.py

```diff
@@ -640,4 +642,64 @@ def test_compute_tfr():
             assert_array_equal(shape[1:], out.shape)
 
 
+@requires_pandas
+def test_getitem_epochsTFR():
+    """Test GetEpochsMixin in the context of EpochsTFR."""
+    from pandas import DataFrame
+    # Setup for reading the raw data and select a few trials
+    raw = read_raw_fif(raw_fname)
+    events = read_events(event_fname)
+    n_events = 10
+
+    # create fake metadata
+    rt = np.random.uniform(size=(n_events,))
```

Comment:
we prefer for fix everywhere the RandomState to avoid random bugs in tests. I know it should not matter here but I prefer we stick to this rule. Please use :

rng = np.random.RandomState(42)
rt = rng.uniform(...)

and remove the calls to np.random below.

thanks